src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -392,6 +392,23 @@
                 });
             }
         },
+        // mounted() {
+        //     let self = this; 
+        //     window.addEventListener('keyup', function(e) {
+        //         if (e.key === 'Enter' && self.projects.length > 0) {
+        //             if (self.renamedProjectName && !self.disableRename && self.projectToRename)
+        //             {
+        //                 self.confirmRename(self.renamedProjectName)
+        //             }
+        //             if (self.newProjectName && !self.disableCreate && self.versionToPromote){
+        //                 self.confirmPromotion(self.newProjectName)
+        //             }
+        //             if (self.projectToDelete || self.versionToDelete){
+        //                 self.confirmDelete()
+        //             }
+        //         }
+        //     });
+        // },
```

Comment:
These captured enter presses for the rename, copyproject/version, and delete project dialogs. I've commented them out as this functionality was being covered by a ticket by Lekan. After merging the latest master, I see enter presses are being registered for rename and copy, but not delete. If delete functionality hasn't been covered I could put a focus on the ok button, similar to how I've done in resetConfirmation, rather than use an event listener.

Reply:
I think this is https://github.com/mrc-ide/hint/pull/465. Given that it has been merged, can the question of outstanding functionality be addressed here?